forge-gui/res/cardsfolder/upcoming/veko_deaths_doorkeeper.txt

```diff
@@ -0,0 +1,11 @@
+Name:Veko, Death's Doorkeeper
+ManaCost:W B
+Types:Legendary Creature Spirit Cleric
+PT:1/3
+K:Extort
+A:AB$ ChangeZone | Cost$ Sac<1/Creature.nonSpirit/nonspirit creature> | RememberChanged$ True | TgtPrompt$ Choose target creature card in your graveyard | ValidTgts$ Creature.YouOwn | Origin$ Graveyard | Destination$ Hand | SorcerySpeed$ True | SubAbility$ DBAnimate | AILogic$ SacAndRetFromGrave | SpellDescription$ Return target creature card from your graveyard to your hand. It perpetually becomes a Spirit, has base power and toughness 1/1, and gains "You may pay {W/B} rather than pay this spell's mana cost." Activate only as a sorcery.
```

Comment:
use AnimateSubAbilty instead of remembering

Reply:
Hmm I don't get this to work with AnimateSubAbility. I guess this isn't wired up outside of battlefield ?

Reply:
without should be fine anyway